web/themes/custom/pineapple/sass/components/fields/_basic-page.scss

```diff
@@ -0,0 +1,53 @@
+.field--name-field-bp-summary {
+  font-size: $medium-font;
+}
+
+.hero-container {
+  height: 527px;
+  overflow: hidden;
+}
+
+.hero-image-container {
+  height: 100%;
+  width: 100%;
+  background-repeat: no-repeat;
+  position: absolute;
+  background-size: cover;
+  background-position: 50%;
+}
+
+.hero-image-text {
+  top: 0;
+  right: 0;
+  bottom: 0;
+  left: 0;
+  position: absolute;
+  display:flex;
+  align-items: flex-end;
+  margin-bottom: 100px;
+
+  .article-title {
+    padding: 15px;
+    background-color: transparentize($primary, 0.4);
+    font-size: $xlarge-font;
+    font-weight: 600;
+    position: absolute;
+    text-align: left;
+    color: $white;
+    max-width: 90%;
+  }
+
+  .author-name {
+    padding: 10px;
+    background-color: transparentize($primary, 0.4);
+    color: $white;
+    text-transform: uppercase;
+    text-align: left;
+    position: absolute;
+    margin-top: 45px;
+
+    p {
+      display: inline;
+    }
+  }
+}
```

Comment:
Hero related code should be nested inside appropriate divs.

Reply:
Also this hero related code (_basic-page.scss) should not be in the "fields" directory, consider moving it to the layouts directory.

Reply:
done